### PR TITLE
LPS-178427 Automate |Setup Automation EVP Project

### PR DESCRIPTION
**Pull request by heiditrindade**
[LPS-178427](https://issues.liferay.com/browse/LPS-178427)

### CODE REVIEW

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			siteTemplateName = "EVP");
+	}
+
+	macro assertPageTitle {
+		AssertElementPresent(
+			key_tabTitle = ${tabTitle},
+			locator1 = "EVPUtils#TAB_TITLE");
+	}
+
+	// @summary = "Go to Folder"
```
Comment (nikki-pru) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro:17]: Please remove comments if the macro is needed.
If the macro is not yet needed, please delete.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+	//macro gotoFolder {
```
Comment (nikki-pru) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro:19]: Please remove comments

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+	macro gotoEVP {
+		Navigator.openSiteURL(siteName = "EVP");
+
+		if (IsElementPresent(locator1 = "EVPUtils#SIGN_IN")) {
+			Click(locator1 = "EVPUtils#SIGN_IN");
+		}
+	}
+
+	// @summary = "Publish an object"
```
Comment (nikki-pru) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro:35]: Please review comments

**Review by nikki-pru — CHANGES_REQUESTED** [review]
@heiditrindade please see changes

**Comment by nikki-pru** [comment]
ci:test:osb-site-initializer-evp

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:osb-site-initializer-evp

**Comment by heiditrindade** [comment]
ci:test:sf
ci:test:osb-site-initializer-evp

**Comment by heiditrindade** [comment]
ci:test:osb-site-initializer-evp

**Comment by nikki-pru** [comment]
ci:test:stable

**Comment by nikki-pru** [comment]
ci:test:stable

**Comment by heiditrindade** [comment]
ci:test:stable

**Comment by heiditrindade** [comment]
ci:test:stable

**Comment by nikki-pru** [comment]
ci:forward

**Comment by heiditrindade** [comment]
ci:test:stable

**Comment by heiditrindade** [comment]
ci:forward

**Comment by heiditrindade** [comment]
ci:forward

**Pull request closed without merging**